src/emails/components/EmailBanner.tsx

```diff
@@ -46,43 +46,41 @@ export const EmailBanner = (props: {
   const LightBanner = (
     <mj-wrapper padding="24px 16px">
       <mj-section padding="24px" background-color="#F9F9FA" border-radius="8px">
-        <mj-wrapper full-width="full-width">
-          <mj-group>
-            <mj-column width="30%" padding-right="16px">
-              <mj-image
-                alt=""
-                src={`${process.env.SERVER_URL}/images/email/monthly-activity/thumbs-up.png`}
-                padding="0"
-                align="left"
-              />
-            </mj-column>
-            <mj-column vertical-align="top" width="70%" padding="0">
-              <mj-text
-                font-size="16px"
-                line-height="24px"
-                padding="0"
-                color="#000000"
-              >
-                <h3>{props.heading}</h3>
-                <p>{props.content}</p>
-              </mj-text>
+        <mj-group>
```

Comment:
@codemist I think the changes to the template are most relevant to you; MJML threw some errors about `<mj-wrapper>`s not being allowed inside sections, and `<mj-group>`s not in wrappers, etc.

(Those errors were not visible in Storybook before; I updated Storybook to also show those errors. I tried to keep the looks the same while fixing the errors.)

Reply:
Thanks for checking! I saw them when actually trying to send the email, which errored out at [the call to `renderEmail`](https://github.com/mozilla/blurts-server/blob/076f70e0f7700a5d3cd99a3f059e557ea709053c/src/scripts/cronjobs/monthlyActivityFree.tsx#L101-L108).

Which of course is a bit cumbersome, which is why I updated the `<StorybookEmailRenderer>` to also show those errors, so that next time, we can see them while actually working on the template :)